blocks/loves/loves.css

```diff
@@ -23,16 +19,16 @@ main .form-container {
   font-style: italic;
 }
 
-.button:hover {
+.loves .button:hover {
   --link-hover-color: rgb(255 255 255);
 }
 
-.button:disabled {
+.loves .button:disabled {
   background: rgb(255 255 255);
-  
+
    --overlay-background-color: rgb(255 255 255);
 }
 
-.button-container {
+.loves .button-container {
   float: right;
```

Comment:
I don't know that you want this float.
![image](https://github.com/mkclawson/impala/assets/3883795/fc99d8ac-6201-404b-a929-d2c221443f78)